src/containers/PipelineRun/PipelineRun.js

```diff
@@ -35,7 +35,7 @@ import { fetchClusterTasks, fetchTasks } from '../../actions/tasks';
 import { fetchTaskRuns } from '../../actions/taskRuns';
 import { rerunPipelineRun } from '../../api';
 
-import '../../components/Run/Run.scss';
+import '@tektoncd/dashboard-components/dist/scss/Run.scss';
```

Comment:
this is already loaded by the PipelineRun component so shouldn't be needed here anymore